app/controllers/merchant/items_controller.rb

```diff
@@ -15,4 +15,13 @@ def update
     end
     redirect_to '/merchant/items'
   end
+
+  def destroy
+    item = Item.find(params[:id])
+    if item.no_orders?
```

Comment:
Is this check different than the one in the view? If we check it once in the view, we wouldn't have to recheck in our destroy method?

Reply:
Plus one to this

Reply:
Okay. Thanks for the check! I will fix it!